apps/new/widget/Index.jsx

```diff
@@ -1 +1,30 @@
-return <p>hello ${config_account}</p>;
+const config = {
+  theme: {},
+  layout: {
+    src: "${alias_devs}/widget/Layout",
+    props: {
+      variant: "standard",
+    },
+  },
+  blocks: {
+    Header: () => <></>,
```

Comment:
Can you initialize with Header as the navbar? Let's go ahead and pull over the right side too

![image](https://github.com/NEARBuilders/gateway/assets/16282460/a453a768-ec91-4582-9a31-6eba6b9cbf12)

Let's not pass in router object to it this time

Reply:
And grab the footer from the landing page

components.header and components.footer? navbar? What do you think, here is the [link to canvas](https://www.tldraw.com/r/rw9W7SsJnejDFsWFznT5K?v=-902,-1781,14400,7940&p=Q2lUnVRyH5IoXJftWXx40)

Reply:
header and footer makes sense

Reply:
we will fork the navbar as header later when we need to make changes